requirements.txt

```diff
@@ -9,11 +9,11 @@ cycler>=0.10.0
 #umap-learn>=0.3
 
 ## Testing Requirements (pip install -r tests/requirements.txt)
-#pytest>=3.4.1
-#pytest-cov>=2.5.1
-#pytest-flakes>=2.0.0
+#pytest>=4.2.0
+#pytest-cov>=2.6.1
+#pytest-flakes>=4.0.0
 #pytest-spec>=1.1.0
```

Comment:
Unfortunately, this is the cause of some of our issues and it hasn't been updated in a while. I'm considering pulling it.

Reply:
Update: I've pulled it while https://github.com/pchomik/pytest-spec/issues/21 is being resolved.

Reply:
Thank you @bbengfort for identifying the trigger for the mysterious testing issues we were experiencing!